### ISSUE TITLE
Firefox for android support 

### ISSUE THREAD

**Issue by midi** [disconnectme/disconnect#334]
Firefox for android users could use an alternative to ghostery.

**Comment by CHEF-KOCH** [comment]
Yep, it would be nice. Currently I use a hosts and copied the needed entries, but a addon to control that like Disconnect is more helpful in this case.

**Comment by jeroenpraat** [comment]
+1 - second that!
